src/modules/voxelDensity/process.cpp

```diff
@@ -0,0 +1,118 @@
+#include "analyser/dataExporter.h"
+#include "analyser/dataOperator1D.h"
+#include "data/atomicMasses.h"
+#include "data/isotopes.h"
+#include "main/dissolve.h"
+#include "math/data3D.h"
+#include "math/matrix3.h"
+#include "module/context.h"
+#include "voxelDensity.h"
+#include <cmath>
+
+Module::ExecutionResult VoxelDensityModule::process(ModuleContext &context)
+{
+    auto &processingData = context.dissolve().processingModuleData();
+
+    // Calculate target property density
+    auto [data3D, status] = processingData.realiseIf<Data3D>(
+        fmt::format("Data3D//{}", targetPropertyTypes().keywordByIndex(static_cast<int>(targetProperty_))), name(),
+        GenericItem::InRestartFileFlag);
+    if (status == GenericItem::ItemStatus::Created)
+    {
+        data3D.initialise(numPoints_);
+    }
+    data3D.zero();
+
+    if (!restrictToSpecies_.empty())
+    {
+        auto cfgSpecies = targetConfiguration_->speciesPopulations();
+
+        for (const auto &[sp, N] : cfgSpecies)
+        {
+            bool included = false;
+
+            for (const auto &spTarget : restrictToSpecies_)
+            {
+                if (sp->name() == spTarget->name())
+                {
+                    included = true;
+                    break;
+                }
+            }
+
+            if (included)
+            {
+                targetConfiguration_->removeMolecules(sp);
+            }
+        }
+    }
+
+    auto unitCell = targetConfiguration_->box();
+    auto voxelVolume = unitCell->volume() / std::pow(numPoints_, 3);
+    auto atoms = targetConfiguration_->atoms();
+
+    auto unaryOp = [this, &data3D, &unitCell](auto &atom)
+    {
+        atom.set(unitCell->foldFrac(atom.r()));
+        auto x = atom.x(), y = atom.y(), z = atom.z();
+        auto atomicNumber = atom.speciesAtom()->Z();
+
+        double value;
+
+        switch (targetProperty_)
+        {
+            case TargetPropertyType::Mass:
+            {
+                // Atomic mass
+                value = AtomicMass::mass(atomicNumber);
+                break;
+            }
+            case TargetPropertyType::AtomicNumber:
+            {
+                // Atomic number
+                value = atomicNumber;
+                break;
+            }
+            case TargetPropertyType::ScatteringLengthDensity:
+            {
+                auto naturalIsotope = Sears91::naturalIsotope(atomicNumber);
+
+                // Bound coherent natural isotope scattering length density
+                value = Sears91::boundCoherent(naturalIsotope);
+                break;
+            }
```

Comment:
```suggestion
            case TargetPropertyType::Mass:
                // Atomic mass
                value = AtomicMass::mass(atomicNumber);
                break;
            case TargetPropertyType::AtomicNumber:
                // Atomic number
                value = atomicNumber;
                break;
            case TargetPropertyType::ScatteringLengthDensity:
                auto naturalIsotope = Sears91::naturalIsotope(atomicNumber);

                // Bound coherent natural isotope scattering length density
                value = Sears91::boundCoherent(naturalIsotope);
                break;
```

Cases for switch statements don't take braces.